test/e2e/resources/rbac-operator.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 ---
 
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
```

Comment:
This file is  not used any where, as we are not using `operator-sdk` for tests.

Reply:
Yes, I missed dropping it in main commit, will do it in this one, I guess.